examples/heartrate_monitor_peripheral.py

```diff
@@ -0,0 +1,117 @@
+"""Example of how to create a Peripheral device/GATT Server"""
+# Standard modules
+import logging
+import struct
+
+# Bluezero modules
+from bluezero import async_tools
+from bluezero import adapter
+from bluezero import peripheral
+
+HRM_SRV = '0000180D-0000-1000-8000-00805f9b34fb'
+HR_MSRMT_UUID =      '00002a37-0000-1000-8000-00805f9b34fb'
+BODY_SNSR_LOC_UUID = '00002a38-0000-1000-8000-00805f9b34fb'
+HR_CTRL_PT_UUID =    '00002a39-0000-1000-8000-00805f9b34fb'
+
+
+heartrate = 60
+def read_heartrate():
+    global heartrate
+    """
+    Example read callback. Value returned needs to a list of bytes/integers
+    in little endian format.
+
+    // Send one byte of flags
+    // As well as an 8 bit measurement
+
+    :return: list of uint8 values
+    """
+    flags = 0b00000110
```

Comment:
This is efficient but maybe a bit cryptic for people not familiar with the Heart Rate service. I don't think it will encourage experimentation.
Any suggestions as to what could be done here? Some kind of pack function? Dataclass?
![image](https://user-images.githubusercontent.com/8608260/172783978-08ad2d43-f317-4b85-bffb-a29855feec83.png)

Reply:
Using IntEnums to include context and not magic numbers.